modules/@ergo/core/components/TemplateGrid/TemplateGridWrapper.vue

```diff
@@ -168,19 +210,18 @@ export default {
             }
         },
         connectionLineStyle({ id, row, parent }) {
+            if (this.constantRoot && row === 0) return { display: 'none' };
             const children = this.filteredGridData.filter(e => e.parent === parent);
             const connectionHeight = this.rowHeight * (
                 row - (children.length ? children[0].row : 0) + 1
             );
             const borderStyle = id === 'ghost_item' ? 'dashed' : 'solid';
             const linePosition = {
```

Comment:
`linePosition` is using **width: linePosition.width,** which is not defined in the function scope.